codegen/settingsgen.py

```diff
@@ -275,12 +351,22 @@ def _populate_classes(parent_dir):
                 pprint.pprint(command_names, stream=strout, compact=True, width=70)
                 mn = ("\n" + istr2).join(strout.getvalue().strip().split("\n"))
                 f.write(f"{istr2}{mn}\n\n")
+                if stubf:
+                    stubf.write(f"{istr1}command_names = ...\n\n")
 
+                commands_info = _get_commands_info(commands_hash)
                 for command in command_names:
-                    f.write(f"{istr1}{command}: {command} = {command}\n")
+                    f.write(f"{istr1}{command}: {command}_cls = {command}_cls\n")
                     f.write(f'{istr1}"""\n')
                     f.write(f"{istr1}{command} command of {cls_name}.")
                     f.write(f'\n{istr1}"""\n')
+                    # function annotation for commands
+                    if stubf:
+                        command_info = commands_info[command]
+                        stubf.write(f"{istr1}def {command}(self, ")
+                        stubf.write(", ".join(command_info.args_info))
+                        stubf.write("):\n")
+                        _write_doc_string(command_info.doc, istr2, stubf)
```

Comment:
Generating typing stubs like
```
def read_case(self, file_type: str, file_name: str, pdf_file_name: str, lightweight_setup: bool):
    """
    'read_case' command.
    
    Parameters
    ----------
        file_type : str
            'file_type' child.
        file_name : str
            'file_name' child.
        pdf_file_name : str
            'pdf_file_name' child.
        lightweight_setup : bool
            'lightweight_setup' child.
    
    """
```
to fix static intellisense. This won't change any runtime behaviour.